server/selection_test.go

```diff
@@ -315,25 +321,60 @@ func TestMinLSSelector_SelectUnknownSession_UniformWeights(t *testing.T) {
 	}
 }
 
-func TestMinLSSelector_SelectUnknownSession_MissingStake(t *testing.T) {
+func TestMinLSSelector_SelectUnknownSession_AllMissingStake(t *testing.T) {
 	assert := assert.New(t)
 
 	stakeRdr := newStubStakeReader()
 	sel := NewMinLSSelector(stakeRdr, 1.0)
 
-	// Initialize stake reader with empty stake map
+	// Initialize stake reader with empty stake map so all sessions are missing stake
 	stakeRdr.SetStakes(make(map[ethcommon.Address]int64))
 
-	sess := &BroadcastSession{
+	addr2 := ethcommon.BytesToAddress([]byte("bar"))
+	sess1 := &BroadcastSession{
```

Comment:
Can we use the same notation here for creating `BroadcastSession`

If it's the same logic we could potentially extract this into a helper

`func createTestBcastSess(recipient ethcommon.Address) *BroadcastSession {
 return &BroadcastSession {
  OrchestratorInfo: &net.OrchestratorInfo {
    TicketParams: &net.TicketParams{ Recipient: recipient.Bytes()}
}
}
}

This would make the code perhaps a little more readable as we're currently starting with an `addr2` declaration but we don't have `addr` declared.

Reply:
Fixed in f24fa47